pkgs/applications/networking/browsers/librewolf/update.nix

```diff
@@ -44,6 +45,13 @@ writeScript "update-librewolf" ''
     exit 1
   fi
 
+  macosAssets=$(curl https://gitlab.com/api/v4/projects/librewolf-community%2Fbrowser%2Fmacos/releases/v$latestTag/assets/links)
```

Comment:
As previously mentioned `librewolf-community` is archived, use `https://gitlab.com/librewolf-community/browser/bsys6/-/archive/$latestTag-1/bsys6-$latestTag-1.tar.gz` instead